### PR TITLE
Eject with adudication reasons

### PR DESCRIPTION
**Pull request by beausmith**
image

image

image

### CODE REVIEW

**Review by eventualbuddha — APPROVED** [review]

File: src/screens/BallotEjectScreen.test.tsx [hunk 1]
```diff
@@ -55,11 +57,9 @@ test('says the ballot sheet is overvoted if it is', async () => {
         metadata: {
           ballotStyleId: '1',
           precinctId: '1',
-          ballotType: BallotType.Standard,
-          electionHash: '',
-          isTestMode: false,
-          locales: { primary: 'en-US' },
+          isTestBallot: false,
           pageNumber: 1,
+          pageCount: 2,
```
Comment (benadida) [inline src/screens/BallotEjectScreen.test.tsx:62]: is this some old code that snuck in there? I think that's the older data model.

**Review by benadida — APPROVED** [review]
some data type stuff to review in the test files, but otherwise looks good to me.

File: src/screens/BallotEjectScreen.test.tsx (hunk 1 above)
Comment (beausmith) [reply src/screens/BallotEjectScreen.test.tsx]: Yes. My bad. Pushing a fix.

Comment (beausmith) [reply src/screens/BallotEjectScreen.test.tsx]: I hadn't run `yarn install` on `bsd`.

**Pull request merged**